source/class/cv/ui/BodyBlocker.js

```diff
@@ -0,0 +1,65 @@
+/* BodyBlocker.js
```

Comment:
What is the purpose of this method?

Reply:
Qooxdoo provides Blockers that overlay certain elements, e.g. add an opacity to the background while you show an popup. The blocker adds the overlay and catches all events to disable any usage of elements in background while the popup is visible.